__tests__/elements/mapchip.test.js

```diff
@@ -0,0 +1,41 @@
+import { screen, fireEvent, getByText } from '@testing-library/dom'
+
+import {MapChip} from "../../src/js/elements/mapchip/mapchip.js";
+import {Component} from '../../src/js/utils';
+
+import html from '../../src/index.html';
+
+describe('Check mapchip HTML description', () => {
+    document.body.innerHTML = html;
+
+    const mapchip_args = {
+        "data": {
+            "description": "<p>An HTML description</p>",
+            "metadata": { 
+                "primary_group": "age group",
+                "groups": []
+            },
+            "chartConfiguration": {
+                "filter": {
+                    "defaults": []
+                }
+            }
+        }
+    }
+    const mapchip_colors = { "colors": [] }
+    let descriptionField;
+
+    beforeEach(() => {
+      descriptionField = document.querySelector('.map-option__context_text');
+    })
+
+    test('Description is visible and renders HTML tags', () => {
+      let component = new Component();
+      let mc = new MapChip(component, mapchip_colors)
+      mc.show();
+      mc.description = mapchip_args.data.description;
+      
+      expect(descriptionField).toBeVisible();
+      expect(descriptionField.textContent.trim()).toBe('An HTML description')
```

Comment:
Does this test if it is displaying html or only text?

If not, perhaps something like
```
...
<p>An <strong>HTML</strong> description</p>
...
htmlTag = descriptionField('strong);
...
expect(htmlTag).not.toBeNull()
```

Reply:
It's checking that the HTML tags are rendered and do no appear the text e.g. <p>An <strong>HTML</strong> description</p> would then just be 'An HTML description'. I did some cleaning of the test, but it's important to check the content of htmlTag after being set

Reply:
but how do you know that it's adding html not escaped text? Wasn't that the bug that was being addressed in the first place?